docs/getting_started.md

```diff
@@ -81,6 +81,37 @@ done
 
 ## Register BYOH host to management cluster
 
+
+### Generating the Bootstrap Kubeconfig file
+Get the APIServer and Certificate Authority Data info
+
+```shell
+APISERVER=$(kubectl config view -ojsonpath='{.clusters[0].cluster.server}')
+CA_CERT=$(kubectl config view --flatten -ojsonpath='{.clusters[0].cluster.certificate-authority-data}')
+```
+
+Create a BootstrapKubeconfig CR as follows
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: v1beta1
+kind: BootstrapKubeconfig
+metadata:
+  name: bootstrap-kubeconfig
+  namespace: default
+spec:
+  apiserver: "$APISERVER"
+  certificate-authority-data: "$CA_CERT"
+EOF
+```
+
+Once the BootstrapKubeconfig CR is created, fetch the object to copy the bootstrap kubeconfig file details from the Status field
+```shell
+kubectl get bootstrapkubeconfig bootstrap-kubeconfig -n default -o json | jq '.status'
+```
+Copy contents into a file called bootstrap-kubeconfig
+
+We need one bootstrap-kubeconfig per host. Create as many bootstrap-kubeconfig files as there are number of hosts (2 for this guide)
+
 ---
 ### VM Prerequisites
```

Comment:
@sachinkumarsingh092 Also, have we documented anywhere about manual approval of certs? IMO, the getting started guide should have automated flow itself. But we also need to mention about manual approval.

Reply:
@anusha94 Currently it seems like it is not possible to provide this via `clusterctl` and the user has to download/update/apply the `infrastructure-components.yaml` for the manual approval of certs. I would suggest taking this in a patch version release. WDYT?
cc: @shivi28 @Madhur97

Reply:
Why do we need to provide via `clusterctl`? This being a feature gate / switch, can we simply ask the user to set the env variable before initializing the provider?

Reply:
Might be missing something but how do you suggest doing this? Users need to provide the environment variable to the manager deployment hence I was thinking to pass it via `clusterctl init`.

Reply:
If it's a feature gate then we can pass it in `~/.cluster-api/clusterctl.yaml` file, in the same fashion as we do for all other feature gates variables. 
> Might be missing something but how do you suggest doing this?
For this we can through an error while creating the workload clusters that we need to set `MANUAL_CSR_APPROVAL ` feature gate. Same thing happens for clusterclass feature

Reply:
[Issue](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/issues/668) created to handle this variable. Resolving this thread for now.